src/scripts/components/dropdown.js

```diff
@@ -53,7 +53,7 @@ export default React.createClass({
 
 	render() {
 		return !this.props.show ? null : (
-			<ul className="dropdown">
+			<ul className={`dropdown ${this.props.class}`}>
```

Comment:
If you add a property, add it to `propTypes`. Also don't use `class` as a property name, use something like `className` instead since it's only reserved for actual DOM elements, not React components.